modules/cluster-resizing/pages/expand-a-cluster.adoc

```diff
@@ -17,6 +17,21 @@ WARNING: If your TigerGraph cluster uses a seeded license, please https://tigerg
 * Obtain a few key measures for the state of your data before the expansion, such as vertex counts/edge counts or certain query results. This will be useful in verifying data integrity after the expansion completes.
 * If the original cluster is a single node installation, make sure the IP used is not a https://en.wikipedia.org/wiki/Localhost[local loopback address] such as 127.0.0.1.
 
+[WARNING]
+====
+If you ever deleted a vertex attribute, cluster expansion might lead to data corruption.
+Prior to expansion, run the following command as the TigerGraph Linux user on your instance to check if you can proceed safely.
+Make sure all TigerGraph services are running when you run the script:
```

Comment:
Maybe explain here how to make sure all services are running? "Use the `gadmin status` command to make sure..."

xref to https://docs.tigergraph.com/tigergraph-server/current/system-management/management-with-gadmin#_gadmin_status